tests/itests-common/src/test/java/org/apache/camel/kafkaconnector/common/utils/NetworkUtils.java

```diff
@@ -29,10 +29,11 @@
 import org.slf4j.LoggerFactory;
 
 public final class NetworkUtils {
-    public static final int  DEFAULT_STARTING_PORT = 49152;
+    
     public static final int  DEFAULT_ENDING_PORT = 65535;
     private static String hostname;
 
+    public static int  DEFAULT_STARTING_PORT = 49152;
```

Comment:
I'd keep `DEFAULT_STARTING_PORT` as `static final int` and declare another `static int startingPort` for actually using it in the code. There is also a style error that all uppercase variables need to be declared before lowercase variables.

Reply:
Thanks @tadayosi for the feedback, will revise accordingly

Reply:
Hi @tadayosi ,

Pushed new commits to address your feedback. 

Thanks!
Freeman